exonum-java-binding-core/rust/ejb-app/modify_runpath.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Modifies RUNPATH of ejb-app executable to make it able to locate java_bindings.so and Rust std lib .so
+# EJB App executable must be in the current directory, dynamic libraries in the `lib` subdirectory.
+
+chrpath -v || { echo "No chrpath installed. Aborting"; exit 1; }
+
+# Convert RPATH to RUNPATH
```

Comment:
We have to do that because the app also needs libjvm on LD_LIBRARY_PATH, which is traversed by the loader _before_ runpath, but after rpath, don't we?

Reply:
Rpath is deprecated, moreover user can change location of libraries usong LD_LIBRARY_PATH